lib/ts/recipe/thirdparty/providers/index.tsx

```diff
@@ -52,7 +55,7 @@ export default abstract class Provider {
         return <ProviderButton logo={this.getLogo()} providerName={providerName} displayName={this.name} />;
     }
 
-    getRedirectURL(): string {
+    defaultGetRedirectURL(): string {
         const domain = SuperTokens.getInstanceOrThrow().appInfo.websiteDomain.getAsStringDangerous();
```

Comment:
Will the routing logic need to change for the SDK to now detect these new paths here: https://github.com/supertokens/supertokens-auth-react/blob/63aa6a289d/lib/ts/recipe/thirdparty/recipe.tsx#L95

Reply:
I think that should be handled by embedding the feature component, as this could be any path (outside of `websiteBasePath` too), and it could even be on an entirely different sub-domain (although that'd require more customization).

Reply:
hmm. okay.